### ISSUE TITLE
Chrome window title not tracked on MacOS

### ISSUE THREAD

**Issue by zplizzi** [ActivityWatch/activitywatch#824]
- [x] I am on the [latest](https://github.com/ActivityWatch/activitywatch/releases/latest) ActivityWatch version.
 - [x] I have searched the issues of this repo and believe that this is not a duplicate.

- **OS name and version**: MacOS 13.0 (Ventura)
- **ActivityWatch version**: 0.12.1

## Describe the bug

Activitywatch never picks up window titles for Chrome. But it does pick up titles for other apps, eg Spotify. I tried to work around this with the web-watcher, but that is unfortunately also broken due to [this bug](https://github.com/ActivityWatch/aw-watcher-web/issues/93).

I don't think this is a duplicate of #785 as I am definitely on 0.12.1.

<img width="490" alt="image" src="https://user-images.githubusercontent.com/5598968/200184546-359f3237-72df-4c7b-b746-cc37e8cca50a.png">

**Comment by zplizzi** [comment]
Nope, although the web-watcher has been stable for me when I installed it from source for some reason.

**Comment by BelKed** [comment]
Maybe it was fixed in the meantime...

If you want, you can download the nightly build from here:
https://github.com/ActivityWatch/activitywatch/actions/runs/3533130809

At the bottom of the page is the *Artifacts* section, where you can find the build for macOS (`builds-macOS-py3.9`) and for other operating systems.

**Comment by rtpHarry** [comment]
I've been experiencing this issue for several months - #787

It was suggested that a pending PR might fix it but that hasn't been merged yet and I don't really want to get involved in building the source or whatnot so at the moment I'm about to roll back to 0.11 which was working fine with window tracking.

The only bad thing about 0.11 was that the tooltip on the timeline got cropped off. I ended up implementing a custom user stylesheet using the chrome plugin Stylus with this content:

```css
.vis-timeline {
    overflow: visible !important;
}
```

**Comment by ErikBjare** [comment]
This should be fixed in the latest beta (v0.12.2b1).
